src/hotspot/cpu/x86/assembler_x86.cpp

```diff
@@ -2564,8 +2627,8 @@ void Assembler::ldmxcsr( Address src) {
   } else {
     NOT_LP64(assert(VM_Version::supports_sse(), ""));
     InstructionMark im(this);
-    prefix(src);
-    emit_int16(0x0F, (unsigned char)0xAE);
+    prefix(src, true /* is_map1 */);
+    emit_int8((unsigned char)0xAE);
     emit_operand(as_Register(2), src, 0);
```

Comment:
Even when UseAVX > 0, if the src address uses higher bank registers, ldmxcsr/stmxcsr should be encoded using the REX2  i.e. the else path.

Reply:
Thank you, fixed.